### ISSUE TITLE
Visualizing a virtuoso graph db

### ISSUE THREAD

**Issue by ghost** [openlink/virtuoso-opensource#666]
This isn't an issue exactly, but I was just wondering if there were any tools to visualize the virtuoso graph database.

**Comment by LLTommy** [comment]
I don't think you are looking for a virtuoso visualisation but a visualisation of the RDF Graph and if you search for that, you will be more successful. Virtuoso is based on SQL, so a virtuoso  visualisation would be like a pictures of tables. Hardly what you are looking for. 

And to (kind of) answer your question, well, there are different things to visualise the results of SPARQL query. To display the structure itself I saw some efforts, nothing that was satisfying for me. It's a bit tricker than it seems. If you find something good, let me know ;)

**Comment by HG1947** [comment]
A small company called https://toucano.uk  have a neat rdf exploration tool which can access any open or basic authentication  endpoint  . The tool runs in javascript . It can draw forced directed graphs from you spraql and is easily configurable . Its like lodlive but you can start with multiple nodes!
